<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>656</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>JPReader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="helperLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="helperFrame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="helperList">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="CSpecTabWidget" name="tabMain">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNewSearch"/>
    <addaction name="separator"/>
    <addaction name="actionWnd"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenClip"/>
    <addaction name="actionOpenInDir"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionRecentDocuments"/>
    <addaction name="separator"/>
    <addaction name="actionPrintPDF"/>
    <addaction name="separator"/>
    <addaction name="actionExitAll"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="actionAddBM"/>
    <addaction name="actionManageBM"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionOpenCB"/>
    <addaction name="actionOpenCBPlain"/>
    <addaction name="separator"/>
    <addaction name="actionClearCB"/>
    <addaction name="actionClearCache"/>
    <addaction name="separator"/>
    <addaction name="actionFindText"/>
   </widget>
   <widget class="QMenu" name="menuCharset">
    <property name="title">
     <string>&amp;Charset</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuTranslationMode">
     <property name="title">
      <string>&amp;Translation mode</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuTranslationLanguages">
     <property name="title">
      <string>Translation languages</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuSubsentencesMode">
     <property name="title">
      <string>Subsentences mode</string>
     </property>
    </widget>
    <addaction name="menuTranslationMode"/>
    <addaction name="menuTranslationLanguages"/>
    <addaction name="menuSubsentencesMode"/>
    <addaction name="separator"/>
    <addaction name="actionTextTranslator"/>
    <addaction name="actionDownloadManager"/>
    <addaction name="actionDictionary"/>
    <addaction name="actionDetachTab"/>
    <addaction name="actionChromiumURLs"/>
    <addaction name="actionTranslatorStatistics"/>
    <addaction name="actionTranslatorCache"/>
    <addaction name="actionWorkerMonitor"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionShowLog"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionSaveSettings"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuCharset"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenInDir">
   <property name="icon">
    <iconset theme="document-open-folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open &amp;in document directory...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenCB">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionClearCB">
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear clipboard</string>
   </property>
  </action>
  <action name="actionOpenClip">
   <property name="icon">
    <iconset theme="document-open-remote">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open with &amp;Url from clipboard</string>
   </property>
  </action>
  <action name="actionWnd">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New &amp;window</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionNewSearch">
   <property name="text">
    <string>New sea&amp;rch</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>S&amp;ave settings</string>
   </property>
  </action>
  <action name="actionAddBM">
   <property name="text">
    <string>&amp;Add bookmark...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionOpenCBPlain">
   <property name="text">
    <string>Paste plain &amp;text</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionTextTranslator">
   <property name="icon">
    <iconset theme="document-edit-verify">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>T&amp;ext translator</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionExitAll">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit and close all</string>
   </property>
  </action>
  <action name="actionDetachTab">
   <property name="icon">
    <iconset theme="tab-detach">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Detach current tab</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="text">
    <string>&amp;Show log</string>
   </property>
  </action>
  <action name="actionFindText">
   <property name="icon">
    <iconset theme="edit-find"/>
   </property>
   <property name="text">
    <string>&amp;Find text</string>
   </property>
  </action>
  <action name="actionDictionary">
   <property name="icon">
    <iconset theme="accessories-dictionary">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>D&amp;ictionary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F9</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionDownloadManager">
   <property name="icon">
    <iconset theme="folder-downloads">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Download &amp;manager</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionRecentDocuments">
   <property name="text">
    <string>Recent &amp;documents</string>
   </property>
  </action>
  <action name="actionClearCache">
   <property name="text">
    <string>C&amp;lear cache</string>
   </property>
  </action>
  <action name="actionPrintPDF">
   <property name="icon">
    <iconset theme="document-print">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Print to PDF...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionManageBM">
   <property name="text">
    <string>&amp;Manage bookmarks</string>
   </property>
  </action>
  <action name="actionChromiumURLs">
   <property name="text">
    <string>Chromium URLs</string>
   </property>
  </action>
  <action name="actionTranslatorStatistics">
   <property name="icon">
    <iconset theme="office-chart-bar">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Translator statistics</string>
   </property>
  </action>
  <action name="actionTranslatorCache">
   <property name="icon">
    <iconset theme="server-database">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Translator cache</string>
   </property>
  </action>
  <action name="actionWorkerMonitor">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Worker monitor</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSpecTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">utils/specwidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
